src/System.Runtime/tests/DBNullTests.InvalidCast.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class DBNullTests
```

Comment:
The problem here is that DBNull does not implement ISerializable and IConvertible yet in ProjectN. It should be like 15 minutes for somebody like @AtsushiKan to fix. It would be much more preferable than refactoring the tests in unnatural ways.